cranelift/entity/src/list.rs

```diff
@@ -458,6 +476,53 @@ impl<T: EntityRef + ReservedValue> EntityList<T> {
         }
     }
 
+    /// Appends another list from the same ListPool onto this list.
+    pub fn append_list(&mut self, other: &EntityList<T>, pool: &mut ListPool<T>) {
+        let offset = self.len(pool);
+        let new_count = other.len(pool);
+        debug_assert!(new_count <= usize::MAX - offset);
+        self.grow(new_count, pool);
+        for i in 0..new_count {
+            let elt = other.get(i, pool).unwrap();
+            *self.get_mut(offset + i, pool).unwrap() = elt;
+        }
+    }
+
+    /// Sorts the list.
+    pub fn sort(&mut self, pool: &mut ListPool<T>)
+    where
+        T: Ord,
+    {
+        self.as_mut_slice(pool).sort_unstable();
+    }
+
+    /// Removes duplicates from the list. Assumes the list has already
+    /// been sorted.
+    pub fn remove_dups(&mut self, pool: &mut ListPool<T>) {
```

Comment:
Should probably be called `dedupe` to match `std::vec::Vec::dedupe`

Reply:
(Removed as per above.)